lib/components/Avatar.js

```diff
@@ -39,7 +39,7 @@ const Avatar = ({
   className = "",
   ...otherProps
 }) => {
-  const [isLoaded, setIsLoaded] = useState(false);
+  const [isLoaded, setIsLoaded] = useState(true);
```

Comment:
why is this change required?

Reply:
@amaldinesh7 this is to render the image initially and fallback to initials on error loading the image. I have updated the condition `shouldDisplayInitials` as well for this.

Reply:
I have refactored the code for the image loading error state.